sdk/program/src/bignum.rs

```diff
@@ -0,0 +1,704 @@
+//! @brief Solana Rust-based BigNumber for bpf-programs
+
+use borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
+use std::fmt;
+
+#[repr(transparent)]
+#[derive(
+    BorshSerialize,
+    BorshDeserialize,
+    BorshSchema,
+    // Clone,
```

Comment:
No?

Reply:
Why is clone commented out?  Do you not want to support the default implementation of `clone`?  If not, remove this line.